Sources/App/Features/Activities/Controllers/ActivityAPIController.swift

```diff
@@ -29,7 +29,7 @@ struct ActivityAPIController: RouteCollection {
         }
 
         // Image upload
-        var fileName = "\(UUID.generateRandom().uuidString)-\(image.activityImage.filename)"
+        let fileName = "\(UUID.generateRandom().uuidString)-\(image.activityImage.filename)"
```

Comment:
I know this change was a bit of a drive by but the warning was super annoying, my bad!